src/rules/at-if-no-null/index.js

```diff
@@ -0,0 +1,44 @@
+import { namespace } from "../../utils";
+import { utils } from "stylelint";
+
+export const ruleName = namespace("at-if-no-null");
+
+export const messages = utils.ruleMessages(ruleName, {
+  equals_null: "Expected @if(statement) rather than @if(statement == null)",
+  not_equals_null: "Expected @if(!statement) rather than @if(statement != null)"
```

Comment:
Are `@if not ($x)` and `@if(!$x)` both valid? Maybe the error messages and tests should have both examples?

Reply:
I changed the messages a little bit. How do they sound now?

Reply:
Sounds good now.